ospfd/ospf_vty.c

```diff
@@ -7236,12 +7243,11 @@ DEFUN (show_ip_ospf_instance_database_max,
 		json = json_object_new_object();
 
 	instance = strtoul(argv[idx_number]->arg, NULL, 10);
-
-	ospf = ospf_lookup_instance(instance);
-	if (ospf == NULL)
+	if (instance != ospf_instance)
 		return CMD_NOT_MY_INSTANCE;
 
-	if (!ospf->oi_running) {
+	ospf = ospf_lookup_instance(instance);
+	if (!ospf || !ospf->oi_running) {
 		vty_out(vty, "%% OSPF instance not found\n");
```

Comment:
I know you didn't change this but I think this message is wrong, right? Can we remove it if you agree?

Reply:
I agree. Removed the log.